Scp096Notifications/Plugin.cs

```diff
@@ -18,14 +18,16 @@ public class Scp096Notifications : Plugin<Config>
 
         public override PluginPriority Priority => PluginPriority.Low;
 
-        public static Scp096Notifications Singleton;
+        public static Plugin Singleton;
         private static EventHandlers Handler;
 
         public override void OnEnabled()
         {
+
             if (!Config.IsEnabled) return;
-            Singleton = new Scp096Notifications();
-            Handler = new EventHandlers();
+            Log.Info($"{Name} plugin has been enabled" + Author);
```

Comment:
Redundant log, base.OnEnabled() does this.

Reply:
(Not sure why it's selecting how it is, I'm just referring to the Log.Info line)

Reply:
oh, its true, wait

Reply:
Ok, I just deleted that line